frame/currency-factory/README.md

```diff
@@ -1,41 +1,51 @@
 # Overview
 
-Currency pallet allows to create new sovereign(local consensus) currencies(assets) from thin air.
+This pallet creates new sovereign(local consensus) currencies(assets) from thin air.
 
-Each currency falls into one of next categories:
+Each currency falls into one of the following categories:
 
 - new currency issued by governance
-- currency produced as consequences of execution of protocol and depends on some base currency(ies), usually named LP
-- currency mapping remote location of asset to local identifiers, foreign currency
+- currency produced as consequences of a protocol execution that depends on one or more base currencies, usually named LP
+- currency mapping the remote location of an asset to local identifiers associated with the foreign currency.
+
+
+Adding well-known identifiers for assets occurs under the following conditions:
+
+- baked into the runtime during compilation
+- added as part of genesis chain specification
+- created in runtime
+
+Not registered in this pallet are compilation time currencies.
   
 ## Basics
 
-Currency must be identified, so factory allows to produce new identifiers.
+We must identify currency. This pallet allows the production of new identifiers to fill that requirement.
 
-Currency to be useful must have amounts. Amounts must be stored on accounts. Accounts must pay ED to prevent spam.
-So each currency must have ED defined.
+A Currency needs an associated amount stored on accounts. Said accounts must pay ED(Existential Deposit) to prevent spam.
+So each currency must have an [ED](../../docs/0002-rent-deposit.md) defined.
 
-In rare edge cases, ED could be zero, but in this case amount is locked into protocol and users cannot freely transfer amounts. In some cases such amounts could be stored in storage without issuing new currency.
+In rare edge cases, the ED could be zero, but in this case, the asset is locked into a protocol, and users cannot freely transfer.
+ We could sometimes store funds without issuing new currency via this factory.
 
 All local currencies are normalized to 12 decimals.
 
 ## Metadata
 
-In some cases governance may add metadata to make currency recognizable, such as:
+In some cases, governance may add metadata to make a currency recognizable, such as:
 
 - name
-- symbol
+- `symbol` Currency may have a human-readable symbol. For example, `XBTC`. This metadata targets for governance to prevent spam and fishing.
```

Comment:
```suggestion
- `symbol`: A currency may have a human-readable symbol. For example, `XBTC`. This metadata targets for governance to prevent spam and fishing.
```

Reply:
Not sure what is meant by "This metadata targets for governance [...]"